drivers/sensor/hts221/hts221.h

```diff
@@ -12,6 +12,9 @@
 #include <zephyr/types.h>
 #include <drivers/gpio.h>
 
+#define HTS221_TRIGGER_ENABLED (DT_INST_NODE_HAS_PROP(0, drdy_gpios) && \
```

Comment:
Shouldn't we add a check to be sure that `drdy_gpios` is present when `CONFIG_HTS221_TRIGGER` is enabled?

Reply:
I don't think so.  The Kconfig signals that support for the trigger should be present in the driver.  That support doesn't require that the signal actually be present in the hardware.  (It might not be in a shield, and the application should get a runtime error if it the code tries to use something that isn't supported by the hardware.)

We probably don't have any policy on that, so I suppose we could start to require that if the driver enables interrupts the hardware must provide support for them.  I don't think that's a good idea, though.

Reply:
ok, txs for feedback